### ISSUE TITLE
Add citation file

### ISSUE THREAD

**Issue by mj-will** [bilby-dev/bilby#844]
We should add a [citation file](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files)

**Comment by adivijaykumar** [comment]
I agree we should add this in. However, were you thinking of just a citation to the bilby paper, or something more comprehensive? In any case I think the cff file should include a link to https://bilby-dev.github.io/bilby/citing-bilby.html at least in a comment.

**Comment by mj-will** [comment]
> I agree we should add this in. However, were you thinking of just a citation to the bilby paper, or something more comprehensive? In any case I think the cff file should include a link to https://bilby-dev.github.io/bilby/citing-bilby.html at least in a comment.

I imagined something more comprehensive, including but not limited to the DOI, author names with ORCIDs and any relevant papers.
